### PR TITLE
t-test power analysis view

### PR DESCRIPTION
**Pull request by hideaki**
# Description
t-test power analysis view

# Checklist
Make sure you have performed the following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/test_wrapper.R [hunk 1]
```diff
@@ -26,6 +26,52 @@ generate_ttest_density_data <- function(t, df, sig_level = 0.05, alternative = "
   ret
 }
 
+generate_ttest_density_data_for_power <- function(d, n1, n2, t, df, sig_level = 0.05, alternative = "two.sided", paired = TRUE) {
```
Comment (kei51e) [inline R/test_wrapper.R:29]: Add a comment for the arguments.

File: tests/testthat/test_ttest_power.R [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  # Case with paired test.
+  mtcars %>% exp_ttest_power(a_ratio = 0.5, paired=TRUE)
+  # Case with "less" alternative.
+  mtcars %>% exp_ttest_power(a_ratio = 0.5, alternative = "less")
+
+  res <- generate_ttest_density_data_for_power(0.2, 100, 100, 2, 100)
+
+  model_df <- exp_ttest_power(mtcars, a_ratio=0.5, d=0.2, sig.level=0.05, beta=0.2, alternative="two.sided", n_start=10, n_end=1000, n_step=10)
+  res <- model_df %>% tidy_rowwise(model, type="summary")
+  expect_equal(colnames(res), c("Probability of Type 1 Error","Probability of Type 2 Error","Power","Effect Size (Cohen's d)","Degree of Freedom","Required Sample Size"))
```
Comment (kei51e) [inline tests/testthat/test_ttest_power.R:14]: Check the output values in addition to the column name if the output is stable and comparable.

File: R/test_wrapper.R (hunk 1 above)
Comment (hideaki) [reply R/test_wrapper.R:29]: Done.

File: tests/testthat/test_ttest_power.R (hunk 2 above)
Comment (hideaki) [reply tests/testthat/test_ttest_power.R:15]: Done.

**Pull request merged**